src/main/java/it/gov/pagopa/payment/service/TransactionService.java

```diff
@@ -1,7 +1,9 @@
 package it.gov.pagopa.payment.service;
 
+
 import it.gov.pagopa.payment.model.TransactionInProgress;
 
 public interface TransactionService {
     TransactionInProgress getTransaction(String id, String userId);
+
```

Comment:
If the only differences on this file are spaces, we can roll it back

Reply:
Given the amount of these occurrences, I would describi why we are asking to you to avoid to commit these changes:

This way our review is full of class changed (actually 16) which not all of them has significant changes